src/js/components/Menu/Menu.js

```diff
@@ -352,7 +353,8 @@ const Menu = forwardRef((props, ref) => {
                   );
                 })}
               </Box>
-              {alignControlMirror === 'bottom' || align.bottom === 'bottom'
+              {!shrunk &&
```

Comment:
Maybe add a comment
// if align.top is defined, don't show the controlMirror when the window height has shrunk

Reply:
Added in for next commit! 😄